nodeletctl/pkg/nodeletctl/nodeletctl.go

```diff
@@ -185,8 +182,8 @@ func DeployCluster(clusterCfg *BootstrapConfig) error {
 			zap.S().Infof("Failed to generate config: %s", err)
 			return fmt.Errorf("Failed to generate config: %s", err)
 		}
-
-		deployer, err := GetNodeletDeployer(clusterCfg, globalClusterStatus, nodeletCfg, nodeletCfg.HostIp, nodeletSrcFile, sshKey)
+		zap.S().Debugf("master nodeletsrc file %s", nodeletSrcFile)
+		deployer, err := GetNodeletDeployer(clusterCfg, globalClusterStatus, nodeletCfg, nodeletCfg.HostIp, nodeletSrcFile, clusterCfg.SSHPrivateKeyFile)
```

Comment:
as mentioned above, don't think we need to pass in clusterCfg.SSHPrivateKeyFile, since clusterCfg is already passed in

Reply:
checking